_topic_map.yml

```diff
@@ -612,6 +612,9 @@ Topics:
 - Name: Syncing LDAP groups
   File: ldap-syncing
   Distros: openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated
+- Name: Creating and using config maps
```

Comment:
This file move was brought up by Siamak. He's correct, config maps docs don't belong in the build bucket.